ui/pages/home/home.component.js

```diff
@@ -205,12 +212,48 @@ export default class Home extends PureComponent {
       setWeb3ShimUsageAlertDismissed,
       originOfCurrentTab,
       disableWeb3ShimUsageAlert,
+      removeSnapError,
```

Comment:
The Snaps-specific imports and functionality should be fenced. See other UI files with fences for examples, e.g. here #12745.